### ISSUE TITLE
Imported config is only available after additional restart

### ISSUE THREAD

**Issue by zjosua** [zjosua/Pokemanki#5]
#### Describe the bug
When legacy config is imported from files to collection, it is only after an additional restart.

#### To Reproduce
1. Remove Pokémanki config: `mw.col.remove_config("pokemanki")` in Anki's debug console
2. Restart Anki
3. Legacy config import tooltip shows, but
  - Deck gears are not replaced with Pokémon images
  - Errors due to missing config pop up when using Pokémanki menus or opening the stats screen

#### Expected behavior
On first restart after deleting the config:
- No errors due to missing config
- Deck gears replaced with Pokémon images

#### What have you tried?
- Delayed init (50 and 100ms)

#### Information about your set-up
```
    Anki 2.1.55 (dev) Python 3.9.10 Qt 6.3.2 PyQt 6.3.1
    Platform: Linux-5.19.10-surface-x86_64-with-glibc2.31
    Flags: frz=False ao=True sv=2
    Add-ons, last update check: 2022-10-04 06:37:11
    
    
    ===Add-ons (active)===
    (add-on provided name [Add-on folder, installed at, version, is config changed])
    '' ['right_hand_reviews', 0, 'None', mod]
    Add Hyperlink ['318752047', 2022-05-23T15:16, 'None', '']
    Anki Simulator ['817108664', 2021-08-07T21:24, 'None', '']
    AnkiWebView Inspector ['31746032', 2022-04-08T21:44, 'None', '']
    Batch Editing ['291119185', 2019-06-17T17:52, 'None', '']
    Cloze Overlapper ['cloze_overlapper', 2019-08-11T08:03, 'None', '']
    Extended Card Stats During Review ['1008566916', 2018-10-03T14:53, 'None', '']
    Extended Schema Hash ['1187737890', 2020-01-21T11:59, 'None', '']
    Field History ['1247884413', 2019-01-08T19:34, 'None', mod]
    Highlight Search Results in the Browser ['highlight_search_results', 2020-05-05T02:19, 'None', '']
    Image Occlusion Enhanced ['image_occlusion_enhanced', 2022-04-09T08:55, 'None', '']
    Leaderboard ['41708974', 2022-05-12T11:46, 'None', mod]
    Mini Format Pack ['mini_format_pack', 2019-08-12T09:37, 'None', '']
    Multiple Choice ['multiple_choice', 2022-04-02T20:20, 'None', '']
    Pokémanki ['pokemanki', 2022-10-02T10:28, 'None', '']
    Pop-up Dictionary ['popup_dictionary', 2019-10-18T20:28, 'None', '']
    Puppy Reinforcement ['puppy_reinforcement', 2019-08-11T08:20, 'None', '']
    Quick tagging 21 ['304770511', 2020-02-11T07:17, 'None', mod]
    Search and Replace Tags ['138501288', 2018-10-01T16:47, 'None', '']
    Spell Check ['spell_check', 2022-09-10T09:39, 'None', '']
    Symbols As You Type ['2040501954', 2022-06-08T06:09, 'None', '']
    Symbols ['1461102936', 2019-06-17T17:52, 'None', mod]
    Visual Feedback for Reviews ['review_feedback', 2020-01-28T21:30, 'None', '']
    gTTS text to speech support ['391644525', 2020-03-15T00:38, 'None', '']
    
    ===IDs of active AnkiWeb add-ons===
    1008566916 1187737890 1247884413 138501288 1461102936 2040501954 291119185 304770511 31746032 318752047 391644525 41708974 817108664
    
    ===Add-ons (inactive)===
    (add-on provided name [Add-on folder, installed at, version, is config changed])
    '' ['print_as_flashcards', 0, 'None', '']
    AMBOSS add-on ['0amboss_addon', 2022-01-19T21:42, 'None', mod]
    Anki Habitica for 21 ['1758045507', 2022-02-09T13:54, 'None', mod]
    Trigger and actions change one card depending on what occurs on another card ['1981494159', 2021-10-18T10:32, 'None', mod]
```

What operating system are you using?
- OS: Debian GNU/Linux
- version: bullseye
